src/Editor/GUI/Editors/EntityEditor.cpp

```diff
@@ -388,7 +388,7 @@ void EntityEditor::ScriptEditor(Component::Script* script) {
         for (size_t i = 0; i != script->refList.size(); ++i) {
             ImGui::Text(script->refList[i]->name.c_str());
             ImGui::SameLine(ImGui::GetWindowWidth() - 30);
-            if(ImGui::SmallButton(("x###remove" + std::to_string(i)).c_str())){
+            if(ImGui::SmallButton(("x###remove" + std::to_string(i)).c_str())) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') horizontal tab used [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AL002)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'if' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)